src/control/destination/background.rs

```diff
@@ -69,16 +70,28 @@ type ClientService = AddOrigin<Backoff<LogErrors<Reconnect<
 /// provided authority to a set of addresses, and ensures that resolution updates are
 /// propagated to all requesters.
 struct Background<T: HttpService<ResponseBody = RecvBody>> {
-    dns_resolver: dns::Resolver,
-    namespaces: Namespaces,
-    destinations: HashMap<DnsNameAndPort, DestinationSet<T>>,
-    /// A queue of authorities that need to be reconnected.
-    reconnects: VecDeque<DnsNameAndPort>,
+    cfg: QueryCfg,
+    dsts: DestinationCache<T>,
+    /// A receiver of new watch requests.
+    request_rx: mpsc::UnboundedReceiver<ResolveRequest>,
```

Comment:
Please try to avoid unnecessary formatting changes like moving variables around. It makes the diff a bit more complicated than it needs to be.

Reply:
Are you referring to how a lot of the fields in this struct were split into two separate structs? In this case, that was actually a functional change --- it was necessary for this code to compile due to borrow checker reasons.